src/Symfony/Component/Form/Tests/Extension/Core/Type/CountryTypeTest.php

```diff
@@ -58,6 +58,25 @@ public function testChoiceTranslationLocaleOption()
         $this->assertContainsEquals(new ChoiceView('MY', 'MY', 'Малайзія'), $choices);
     }
 
+    /**
+     * @requires extension intl
+     */
+    public function testAlpha3Option()
```

Comment:
perhaps we should add a 2nd test (testAlpha3OptionWithTranslationLocale) to ensure alpha3 codes are translatable as well. See inspiration above.

Reply:
I've added it!